boltdesignsystem/bolt
# DS-486: Jest Test Button/Link Update

## Jira

[https://pegadigitalit.atlassian.net/browse/DS-486](https://pegadigitalit.atlassian.net/browse/DS-486)

## Summary

Simplify component tests, update element tests.

## Details

We're working toward removing the button and link components, so we're simplifying the tests for those. I have also updated the tests for the button and link elements as we move toward using those more.

## How to test

Check out the code and see if we're testing what we want to, Travis should handle the rest.
